engine/src/main/java/org/terasology/rendering/openvrprovider/OpenVRState.java

```diff
@@ -55,10 +55,34 @@
         }
     }
 
+    /**
+     * Add a controller listener. This listener will receive pose and button state updates for the controller.
+     * @param toAdd - the controller listener. Implements the ControllerListener interface.
```

Comment:
I'd write: 

`@param listener - an object implementing the ControllerListener interface.`

And yes, I recommend a change of variable name.

This way the code becomes `controllerListeners.add(listener);` which is marginally closer to plain English.